erpc.yaml

```diff
@@ -40,34 +40,58 @@ projects:
         - 'OPTIONS'
 
     networkDefaults:
+      directiveDefaults:
+        retryPending: false
+        retryEmpty: true
       failsafe:
-        circuitBreaker: ~
         timeout:
-          duration: 60s
-        retry:
-          maxAttempts: 3
-          delay: 500ms
-          backoffMaxDelay: 10s
-          backoffFactor: 0.3
-          jitter: 200ms
+          duration: 300s
         hedge:
-          delay: 1000ms
-          maxCount: 2
+          quantile: 0.95
+          maxCount: 3
+          minDelay: 100ms
+          maxDelay: 5s
```

Comment:
With only 1 upstream let's move this to upstream level with quantile 0.99 and maxCount 1 to just cover intermittent network glitches